jenkins/nephos/buildimage-nps-all/Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+pipeline {
```

Comment:
This pipeline will be as a replacement for https://sonic-jenkins.westus2.cloudapp.azure.com/job/nephos/job/buildimage-nephos-all/, right?

Reply:
yes,that is right